python/ray/_private/ray_logging.py

```diff
@@ -354,10 +354,13 @@ def deduplicate(self, batch: LogBatch) -> List[LogBatch]:
             elif self.skip_re and self.skip_re.search(line):
                 continue
             dedup_key = _canonicalise_log_line(line)
+
             if dedup_key in self.recent:
                 sources = self.recent[dedup_key].sources
                 sources.add(source)
-                if len(sources) > 1:
+                # error_event doesn't report sources.
+                # So we don't have the restriction.
+                if len(sources) > 1 or batch["pid"] == "error_event":
```

Comment:
this is a hack. Since the error pubsub doesn't include the IP, the source is always the same (1 source), so it is never deduplicatd. Q: should we just report IP address?

Reply:
How about we set the pid to "raylet" and remove that from the `should_dedup` check on worker.py:1760? Since this is emitted from raylet.